staging/operator-registry/pkg/lib/indexer/indexer_test.go

```diff
@@ -83,3 +83,61 @@ func TestGeneratePackageYaml(t *testing.T) {
 
 	_ = os.RemoveAll("./package.yaml")
 }
+
+func TestBuildContext(t *testing.T) {
+	// TODO(): Test does not currently have a clean way
```

Comment:
those functions could be improved by using the new `fs.FS` abstractions -- then it would be easier to test